packages/warehouses/src/warehouseClients/BigqueryWarehouseClient.ts

```diff
@@ -220,14 +220,16 @@ export class BigqueryWarehouseClient extends WarehouseBaseClient<CreateBigqueryC
 
             await streamPromise;
         } catch (e: unknown) {
-            const isIJob = (error: unknown): error is bigquery.IJob =>
+            type BigqueryError = {
```

Comment:
bigquery errors look like this 
![image](https://github.com/user-attachments/assets/4575b1a8-8720-4971-837d-82fd8be27534)